### DISCUSSION TITLE
Class "Filament\Support\Assets\AlpineComponent" not found

### DISCUSSION THREAD

**Discussion by atenadadkhah** [awcodes/filament-tiptap-editor#297] [Q&A]
I just installed v3, but when I wanted to run my app, I saw this error:

`Class "Filament\Support\Assets\AlpineComponent" not found` 

I think it's coming from FilamentTiptapEditorServiceProvider.php file.

I tried removing and reinstalling the package many times, but didn't work.

What is the problem for?

**Comment by awcodes** [comment]
Sounds like you are not actually on Filament v3.

You can run `php artisan about` and it will tell you what version is actually installed to the app.

**Comment by atenadadkhah** [comment]
Yes, but in fact I can't update filament to v3 because my app is a bit old, but I really need this feature in the editor, what can I do?

I just want to be able to link a headline or text to a part of the text, so is there any other way than "table of content" ?

**Comment by awcodes** [comment]
Ah. V3 only works with Filament v3. You'll need to install v2 of the plugin.

**Reply by atenadadkhah** [reply]
Is there any way to have this table of content in v2 ? Cause it's necessary for me to have a table of content in the editor

**Reply by awcodes** [reply]
In v2 no. I have no plans to back port features into v2 especially since I've started working on v4.
